playground/index.html

```diff
@@ -741,5 +753,36 @@
     <script src="../vm.worker.js"></script>
     <!-- Playground -->
     <script src="./playground.js"></script>
+    <script>
+        function toXml() {
+            var output = document.getElementById('importExport');
+            var xml = Blockly.Xml.workspaceToDom(workspace);
+            output.value = Blockly.Xml.domToPrettyText(xml);
+            output.focus();
+            output.select();
+            taChange();
+          }
+    
+          function fromXml() {
+            var input = document.getElementById('importExport');
+            var xml = Blockly.Xml.textToDom(input.value);
+            Blockly.Xml.domToWorkspace(workspace, xml);
+            taChange();
+          }
+    
+          // Disable the "Import from XML" button if the XML is invalid.
+          // Preserve text between page reloads.
+          function taChange() {
```

Comment:
For the sake of simplicity, can we cut out `taChange` and the calls to it above (in `toXml`, `fromXml`, and the `onchange` and `onkeyup` handlers in the HTML?)